pkg/db/src/types.ts

```diff
@@ -103,11 +98,15 @@ export interface NoteProto<A extends Record<string, any> = {}> {
 	/** Set name updates the `displayName` of the note. */
 	setName: (name: string, ctx: debug.DebugCtx) => Promise<NoteInterface<A>>;
 	/**
-	 * Add volumes accepts an array of volume stock transactions and adds them to the note.
-	 * If any transactions (for a given isbn/warehouse) already exist, the quantity gets aggregated.
+	 * Add volumes accepts an array of volume stock entries and adds them to the note.
+	 * If any transactions (for a given isbn and warehouse) already exist, the quantity gets aggregated.
+	 */
+	addVolumes: (...params: VolumeStock[]) => Promise<NoteInterface<A>>;
+	/** Explicitly update an existing transaction row.
+	 * the transaction is matched with both isbn and warehouseId.
+	 * If entry with the same isbn previously has no warehouseId and
+	 * a warehouseId was provided, the empty warehouseId will be overwritten
```

Comment:
I wouldn't bother with non existing warehouse id being updates, we can do it as a second pass, let's, for now just match by isbn and warehouseId (as is).

Also, multiline JSDoc should start on the second line (not after `/**`)